zenith/src/main.rs

```diff
@@ -445,10 +441,8 @@ fn handle_tenant(tenant_match: &ArgMatches, env: &local_env::LocalEnv) -> Result
             pageserver.tenant_create(tenantid)?;
             println!("tenant successfully created on the pageserver");
         }
-
-        (sub_name, _) => {
-            bail!("Unexpected tenant subcommand '{}'", sub_name)
-        }
+        Some((sub_name, _)) => bail!("Unexpected tenant subcommand '{}'", sub_name),
+        None => bail!("No tenant subcommand found"),
```

Comment:
The error messages on missing subcommand are inconsistent:

No tenant subcommand found
no pg subcommand provided
No pageserver subcommand given
no safekeeper subcommand provided

Let's standardize on some variant. I like the "no X subcommand provided" the best. Not sure if it should be capitalized or not, but let's try to be consistent on that too.